benchmark/custom_csv_formater.hpp

```diff
@@ -0,0 +1,181 @@
+#pragma once
```

Comment:
I gave this a try and everything seems to work great! There's just one more little thing I noticed that I missed on the first pass yesterday (sorry) - I think this file is missing the license/copyright description at the top. You should be able to just copy/paste it from another header. Just need to make sure the copyright year is up to date (2024).